.github/workflows/gerrit-compose-required-maven-nexus-iq.yaml

```diff
@@ -0,0 +1,109 @@
+        required: true
+        type: string
+      GERRIT_CHANGE_ID:
+        description: "The ID for the change"
+        required: true
+        type: string
+      GERRIT_CHANGE_NUMBER:
+        description: "The Gerrit number"
+        required: true
+        type: string
+      GERRIT_CHANGE_URL:
+        description: "URL to the change"
+        required: true
+        type: string
+      GERRIT_EVENT_TYPE:
+        description: "Type of Gerrit event"
+        required: true
+        type: string
+      GERRIT_PATCHSET_NUMBER:
+        description: "The patch number for the change"
+        required: true
+        type: string
+      GERRIT_PATCHSET_REVISION:
+        description: "The revision sha"
+        required: true
+        type: string
+      GERRIT_PROJECT:
+        description: "Project in Gerrit"
+        required: true
+        type: string
+      GERRIT_REFSPEC:
+        description: "Gerrit refspec of change"
+        required: true
+        type: string
+      TARGET_REPO:
+        # yamllint disable-line rule:line-length
+        description: "The target GitHub repository needing the required workflow"
+        required: false
+        default: ${{ github.repository }}
+        type: string
+    secrets:
+      NEXUS_IQ_PASSWORD:
+        description: "CLM Nexus token"
+        required: true
+
+concurrency:
+  # yamllint disable-line rule:line-length
+  group: compose-maven-nexus-iq-${{ github.workflow }}-${{ github.event.inputs.GERRIT_CHANGE_ID || github.run_id }}
+  cancel-in-progress: true
+
+env:
+  APPLICATION_ID: ${{ vars.ORGANIZATION }}_${{ inputs.TARGET_REPO }}
+  NEXUS_IQ_SERVER: https://nexus-iq.wl.linuxfoundation.org
+  NEXUS_STAGE: build
+  NEXUS_TARGET: ./target/
+
+jobs:
+  run-maven-clm:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.GERRIT_BRANCH }}
+          repository: ${{ inputs.TARGET_REPO }}
+          submodules: "true"
+      - uses: actions/setup-python@v4
+        id: setup-python
+        with:
+          python-version: "3.8"
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Set up Maven
+        uses: thermofisher/setup-maven@v1.0.0
+        with:
+          maven-version: 3.5.4
+      - name: Obtain global settings file
+        # yamllint disable rule:line-length
+        run: |
+          if [[ "${{ vars.ORGANIZATION }}" == "opendaylight" ]]; then
+              wget -q -O settings.xml "https://raw.githubusercontent.com/${{ vars.ORGANIZATION }}/releng-builder/master/jenkins-config/managed-config-files/globalMavenSettings/global-settings/content"
+          else
+              wget -q -O settings.xml "https://raw.githubusercontent.com/${{ vars.ORGANIZATION }}/ci-management/master/jenkins-config/managed-config-files/globalMavenSettings/global-settings/content"
+          fi
+      - name: Build code with Maven
```

Comment:
You need to add `# yamllint enable rule:line-length` after the run block

Reply:
done